.github/workflows/test.yml

```diff
@@ -152,6 +152,7 @@ jobs:
           RECORD_REPLAY_WEBHOOK_URL: ${{ secrets.RECORD_REPLAY_WEBHOOK_URL }}
           RECORD_REPLAY_TEST_METRICS: 1
           AUTOMATED_TEST_SECRET: ${{ secrets.AUTOMATED_TEST_SECRET }}
+          AUTHENTICATED_TESTS_WORKSPACE_API_KEY: ${{ secrets.AUTHENTICATED_TESTS_WORKSPACE_API_KEY }}
```

Comment:
We'll need to make sure to add a secret for this in the backend repo too.

Reply:
https://github.com/replayio/backend/pull/8235

Reply:
Can you please set in devtools repo @bvaughn?